gwpy/types/tests/test_array2d.py

```diff
@@ -216,6 +216,16 @@ def test_single_column_slice(self):
         utils.assert_array_equal(b.xindex, a.xindex[0:1])
         utils.assert_array_equal(b.yindex, a.yindex)
 
+    def test_two_index_arrays(self):
+        """Test that subsetting with two index arrays works correctly."""
```

Comment:
Just to match the rest of the code:

```suggestion
        """Test that subsetting with two index arrays works correctly.
        """
```